src/ArborX_DistributedSearchTree.hpp

```diff
@@ -105,9 +105,10 @@ template <typename DeviceType>
 template <typename Primitives>
 DistributedSearchTree<DeviceType>::DistributedSearchTree(
     MPI_Comm comm, Primitives const &primitives)
-    : _comm(comm)
-    , _bottom_tree(primitives)
+    : _bottom_tree(primitives)
 {
+  MPI_Comm_dup(comm, &_comm);
```

Comment:
You need to comment about the duplication.  I do not understand what is different from copying since it duplicate the attributes of the original communicator.

Reply:
Copying communicator does not create new context. Duplicating does. Having a different context means that the messages sent by user cannot be received by the library and vice versa, even if they have the exact same information and tags.

Reply:
Add a comment in the code.